plottable.css

```diff
@@ -43,6 +43,10 @@ svg.plottable {
   display : block; /* SVGs must be block elements for width/height calculations to work in Firefox. */
 }
 
+.plottable .background-fill {
```

Comment:
We should probably think of what to name this background rectangle as it easily has more capability can just coloring the fill